### PR TITLE
fix: add router_basename for models with compound words

### PR DESCRIPTION
**Pull request by bzwei**
It helps to reverse the url to access the resource

Needed by https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/lib/abstract_models/common.py#L21

AAP-25487: Role user assignment response missing content_object except for Activations

### CODE REVIEW

**Review by Alex-Izquierdo — COMMENTED** [review]
Why shouldn't we have this field defined in all our models?

**Comment by bzwei** [comment]
> Why shouldn't we have this field defined in all our exposed models?
Adding this field to single word models does not change the resulting base name, but yes we can do this to all exposed models.

File: src/aap_eda/core/models/credential_type.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
 
 class CredentialType(BaseOrgModel, UniqueNamedModel):
+    router_basename = "credentialtype"
```
Comment (mkanoor) [inline src/aap_eda/core/models/credential_type.py:23]: @bzwei Should this be a meta property?

Comment (bzwei) [reply src/aap_eda/core/models/credential_type.py:23]: no. it is a class variable. I followed the example of DAB

**Review by mkanoor — APPROVED** [review]

**Review by hsong-rh — APPROVED** [review]

**Pull request merged**